src/client/signatures/canonical_json/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "canonical_json"
+version = "0.1.0"
+authors = ["Vishwa Patel <vpatel@mozilla.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+log = "0.4.0"
+env_logger = "0.7.1"
```

Comment:
This should be a dev dependency.

Reply:
yes, will make this change in the main crate as well